master.xml

```diff
@@ -17,20 +17,20 @@
   
   <default remote="seL4" revision="master"/>
   
-  <project name="musllibc.git" path="projects/musllibc" remote="seL4" revision="4a8335b2248d934e2e40386af4f1b0495b3c173d" upstream="sel4"/>
+  <project name="musllibc.git" path="projects/musllibc" remote="seL4" revision="3d6b939e8f05cb1d2a1a8c8166609bf2e652e975" upstream="sel4"/>
```

Comment:
If we agree on the branch `riscv_hyp`, we could also reference it here instead of a commit ID, so it would track the work in progress.